deployment/deploy-app.ps1

```diff
@@ -32,7 +32,7 @@ $startTime = Get-Date
 # ----- Deploy Bicep
 Write-Title("Deploy Bicep File")
 $r = (az deployment sub create --location $Location `
-           --template-file .\bicep\iiot-app.bicep --parameters applicationName=$ApplicationName `
+           --template-file .\bicep\iiot-app.bicep --parameters applicationName=$ApplicationName aksObjectId="" acrCreate=false `
```

Comment:
I think if we have default value of '' for askObjectId and false for acrCreate in iiot-app.bicep, we don't need to provide values here?

Reply:
That will be cleaner, will update.